cypress/e2e/api/posts/getList.spec.js

```diff
@@ -0,0 +1,8 @@
+describe("Posts", ()=>{
+    it('should return list of posts', () => {
+        cy.api({
+            url: '/posts',
+            method: 'GET',
+        }).its("body").should("have.length", 100)
```

Comment:
also check status code and items structure (each posy should have id, tittle, body, userId )